Modules/mmapmodule.c

```diff
@@ -703,7 +703,7 @@ mmap__sizeof__method(mmap_object *self, void *unused)
 
     res = _PyObject_SIZE(Py_TYPE(self));
     if (self->tagname)
-        res += strlen(self->tagname) + 1;
+        res += wcslen(self->tagname) + 1;
```

Comment:
The `__sizeof__` method is calculating the size in bytes of the object, but `wcslen` will return the number of characters.
```suggestion
        res += (wcslen(self->tagname) + 1) * sizeof(self->tagname[0]);
```

Reply:
I'm sorry. You're right.